### PR TITLE
xz: add page

### PR DESCRIPTION
**Pull request by tonytheleg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
xz (XZ Utils) 5.2.5
liblzma 5.2.5

Addresses #8859

### CODE REVIEW

**Comment by tonytheleg** [comment]
I almost meant to mention in the PR
`unxz`, `xzcat`, `lzma`, `unlzma`, and `lzcat` are all aliases for some type of `xz` command with flags. The rest of the commands in the list are each their own with their own help screens and probably warrant tldr pages. Thanks!

File: pages/linux/xz.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# xz
+
+> Data compression software with a high compression ratio.
+> Can compress typical files ~30% smaller than gzip and ~15% smaller output than bzip2.
+> Syntax is similar to that of gzip.
+> More information: <https://tukaani.org/xz/>.
+
+- Compress a single file:
+
+`xz {{file}}`
```
Comment (einverne) [inline pages/linux/xz.md:10]:
```suggestion
`xz {{path/to/file}}`
```

File: pages/linux/xz.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- Compress multiple files:
+
+`xz {{file1}} {{file2}} {{file3}}`
```
Comment (einverne) [inline pages/linux/xz.md:14]:
```suggestion
`xz {{path/to/file1}} {{path/to/file2}} {{path/to/file3}}`
```

File: pages/linux/xz.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+- Decompress a file:
+
+`xz -d {{file.xz}}`
```
Comment (einverne) [inline pages/linux/xz.md:18]:
```suggestion
`xz -d {{path/to/file.xz}}`
```

File: pages/linux/xz.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+- Keep the input file (by default xz replaces the input file with the compressed/decompressed file):
+
+`xz -k {{file}}`
```
Comment (einverne) [inline pages/linux/xz.md:22]:
```suggestion
`xz -k {{path/to/file}}`
```

File: pages/linux/xz.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+- Write the compressed or decompressed data to standard output:
+
+`xz -cd {{file}}`
```
Comment (einverne) [inline pages/linux/xz.md:26]:
```suggestion
`xz -cd {{path/to/file}}`
```

File: pages/linux/xz.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+- Set the compression level (0 to 9, default is 6):
+
+`xz {{-9}} {{file}}`
```
Comment (einverne) [inline pages/linux/xz.md:30]:
```suggestion
`xz {{-9}} {{path/to/file}}`
```

**Review by einverne — CHANGES_REQUESTED** [review]
@tonytheleg Thank you for the new page, I left a suggestion for you.

**Comment by tonytheleg** [comment]
@einverne great suggestion, just added those changes. Thanks!

File: pages/linux/xz.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# xz
+
+> Data compression software with a high compression ratio.
+> Can compress typical files ~30% smaller than gzip and ~15% smaller output than bzip2.
+> Syntax is similar to that of gzip.
```
Comment (kbdharun) [inline pages/linux/xz.md:5]: I have a doubt can pages with 4 lines of page description be created? Would there be any linter errors? @navarroaxel

File: pages/linux/xz.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+> More information: <https://tukaani.org/xz/>.
```
Comment (kbdharun) [inline pages/linux/xz.md:6]: I couldn't find the documentation at this link. Can you update this one with a more specific one?

**Comment by kbdharun** [comment]
> Hello! I've noticed something unusual when checking this PR:
> 
> * The page `linux/xz.md` already exists under the `common` platform.
> 
> Is this intended? If so, just ignore this comment. Otherwise, please double-check the commits.

Regarding this, Should we update the page in common with this?

File: pages/linux/xz.md (hunk 8 above)
Comment (tonytheleg) [reply pages/linux/xz.md:6]: Hi @kbdharun I couldn't find any documentation provided by them directly, most of this info was grabbed from man pages directly. If you want I can update with a link to a man page site as that's probably more useful to users
https://manpages.org/xz

Comment (kbdharun) [reply pages/linux/xz.md:6]: > Hi @kbdharun I couldn't find any documentation provided by them directly, most of this info was grabbed from man pages directly. If you want I can update with a link to a man page site as that's probably more useful to users https://manpages.org/xz

That would be great, we generally prefer manned for man pages.

Comment (tonytheleg) [reply pages/linux/xz.md:6]: Will do, Ill hold off on the change until the other comments are confirmed and get them all submitted together. Thanks!

**Comment by tonytheleg** [comment]
@kbdharun I didn't see a PR linked in the original issue but looking back at it now, it looks like `xz` is checked off the list which the bot did catch. Since there is already a translated page duplicating that one, this PR is probably extra and unneeded. I can grab some other ones on the list to help out. Thanks

File: pages/linux/xz.md (hunk 7 above)
Comment (navarroaxel) [reply pages/linux/xz.md:5]: 3 lines max

Comment (kbdharun) [reply pages/linux/xz.md:5]: I am not sure why tldr-bot didn't give an error. Wouldn't it recognise this?

**Comment by pixelcmtd** [comment]
> I didn't see a PR linked in the original issue[.] Since there is already a translated page duplicating that one, this PR is probably extra and unneeded.

@tonytheleg That was an oversight of both the original author of #8859 and us looking at it, apologies for your wasted time on behalf of everyone involved.

> I can grab some other ones on the list to help out.

That would be very much appreciated, thank you.

**Pull request closed without merging**